### PR TITLE
Add downgrade language to the dapr upgrade cli docs

### PR DESCRIPTION
**Pull request by greenie-msft**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [x] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description
Added language to the CLI 'upgrade' docs that the command can also be used to downgrade or rollback a version bump. 

Note: Would like to see a separate command for "downgrading" offered in the CLI.

### CODE REVIEW

**Comment by orizohar** [comment]
@artursouza this looks good to me, can you take a look as well?

File: daprdocs/content/en/reference/cli/dapr-upgrade.md [hunk 1]
```diff
@@ -7,7 +7,11 @@ description: "Detailed information on the upgrade CLI command"
 
 ## Description
 
-Upgrade Dapr on supported hosting platforms.
+Upgrade or downgrade Dapr on supported hosting platforms. 
+
+> Note: Version steps should be done incrementally, including minor versions as you upgrade or downgrade.
+
+> Note: Prior to downgrading, confirm components are backwards compatible and application code does ultilize APIs that are not supported in previous versions of Dapr.
```
Comment (AaronCrawfis) [inline daprdocs/content/en/reference/cli/dapr-upgrade.md:14]: Pls use Docsy alert shortcode: https://www.docsy.dev/docs/adding-content/shortcodes/#alert

**Comment by orizohar** [comment]
We should ensure this language is also part of the CLI prints for the command

**Comment by greenie-msft** [comment]
Closing this PR and opening a new one since GH is encountering issues and latest commits are not showing up

**Pull request closed without merging**